src/keboola_mcp_server/clients/data_science.py

```
@@ -0,0 +1,252 @@
+        _authorization = DataAppConfig.Authorization.model_validate(authorization).model_dump(
+            exclude_none=True, by_alias=True
+        )
+        data = {
+            'branchId': None,
+            'name': name,
+            'type': 'streamlit',
+            'description': description,
+            'config': {
+                'parameters': _params,
+                'authorization': _authorization,
+            },
+        }
+        response = await self.post(endpoint='apps', data=data)
+        return DataAppResponse.model_validate(response)
+
+    async def delete_data_app(self, data_app_id: str) -> None:
+        """
+        Delete a data app by its ID.
+        """
+        await self.delete(endpoint=f'apps/{data_app_id}')
+
+    async def list_data_apps(self, limit: int = 100, offset: int = 0) -> list[DataAppResponse]:
+        """
+        List all data apps.
+        """
+        response = await self.get(endpoint='apps', params={'limit': limit, 'offset': offset})
+        return [DataAppResponse.model_validate(app) for app in response]
+
+    async def tail_app_logs(self, app_id: str, since: Optional[str] = None, *, lines: Optional[int] = None) -> str:
+        """
+        Tail application logs. Either `since` or `lines` must be provided but not both at the same time, otherwise it
+        uses the `lines` parameter. In case when none of the parameters are provided, it uses the `lines` parameter with
+        the last 100 lines.
+        :param app_id: ID of the app.
+        :param since: ISO-8601 timestamp with nanoseconds.
+                      E.g: since = (datetime.now(timezone.utc) - timedelta(days=1)).isoformat()
+        :param lines: Number of log lines from the end. Defaults to 100.
+        :return: Logs as plain text.
+        :raise requests.HTTPError: For non-200 status codes.
+        """
+        if since and lines:
```

Maybe we should rather throw an error in both cases below